config/jobs/kubernetes/sig-release/kubernetes-builds.yaml

```diff
@@ -48,9 +48,12 @@ periodics:
       securityContext:
         privileged: true
       resources:
+        limits:
+          cpu: 8
```

Comment:
This will likely cause pod scheduling failure every time as there is overhead for a few other processes on a node, suggest something like `7300m` (ref: https://github.com/kubernetes/test-infra/pull/18420)

Reply:
the values are different for this cluster, because the build jobs are still in google.com prow, and cannot easily be moved (due to the release bucket)

Reply:
@BenTheElder ah gotcha, I wonder if we could add some of the basic stats of that cluster to the umbrella issue